### ISSUE TITLE
Unable to verify contract in phase 2 when providing .sol and metadata files - same process produces perfect match in phase 1

### ISSUE THREAD

**Issue by ed-marquez** [hashgraph/hedera-mirror-node-explorer#755]
### Description
Tried verifying smart contracts with these tool combinations:
- `solc` (not solcjs) for generating abi, bytecode, metadata JSON + ethers js for deployment
- Remix for generating abi, bytecode, metadata JSON + ethers js for deployment
 
Both tool combinations produced the same exact results shown below.

### Flow
For the same contract:
**Phase 1**:
- Requires both the .sol and metadata JSON
- When both files are provided, verification results in a perfect match
- Verification not possible with just 1 file (either one)

**Phase 2:**
- If both the .sol and metadata JSON are provided, verification fails with `Resource missing; unsuccessful fetching: contracts/InventoryLookup.sol`
- Providing only the metadata doesn’t enable the verify button.
- Providing only the .sol results in a partial match

The main issue is that, for phases 1 and 2, the same inputs produce different/inconsistent results.

https://github.com/hashgraph/hedera-mirror-node-explorer/assets/72571340/43941d79-e1e7-4df7-bcbc-af8b2e8483f5

### Steps to reproduce

[GitHub repo](https://github.com/ed-marquez/hedera-example-smart-contract-verification-ethersjs-solc) with the example code
- [Try in GitPod](https://gitpod.io/#https://github.com/ed-marquez/hedera-example-smart-contract-verification-ethersjs-solc) for convenience

### Hedera network

testnet

### Version

https://hashscan-latest.hedera-devops.com/testnet/dashboard

### Operating system

macOS

**Comment by ed-marquez** [comment]
The bug may still exist (for Remix only)
Looks like there may be an operating system factor at play here... 


I did all my initial testing on a Mac. I saw this:

- .sol and metadata are required for a full match
- .sol file only will get you to a partial match


Now, I'm creating content on my work Windows PC... I'm seeing that:
**Using Remix for compilation,**

- Providing the metadata JSON alone does not enable the verify button (Good - consistent with what I had seen before)
- Providing the .sol file alone reaches partial match (Good - consistent with what I had seen before)
- Providing both the .sol and metadata JSON files (required for full match) is failing the verification. This is true for initial verification and re-verification after a partial match (img 1 and 2). This same flow works ok on Mac... 

![image](https://github.com/hashgraph/hedera-mirror-node-explorer/assets/72571340/a51c12ed-9f56-4a65-bcb4-feeb92800307)
![image](https://github.com/hashgraph/hedera-mirror-node-explorer/assets/72571340/7e7a7b84-78de-4c46-aed6-19969ba8b9ec)
